src/cryptsetup/cryptsetup-tokens/luks2-tpm2.c

```diff
@@ -58,7 +58,7 @@ int acquire_luks2_key(
         if ((flags & TPM2_FLAGS_USE_PIN) && salt && !pin)
```

Comment:
Otherwise, I guess we should also update the condition here.

Reply:
No, it is ok if the salt is missing, it was added later to the token

Reply:
@poettering What do you think about the line?